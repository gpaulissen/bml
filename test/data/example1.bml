1C  2+!c. Natural or balanced
1D  4+ suit, unbalanced
1M  5+ M, showing a Major
2C  20--21 bal / Any game force
2D  6+!h or 6+!s, 5--9 hcp
2HS 6+ suit, 10--13 hcp
2N  22--24
3X  6+X, Preemptive
3N  Gambling
