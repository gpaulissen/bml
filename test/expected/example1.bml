#+TITLE: 

#+DESCRIPTION: 

#VUL 00

#SEAT 0

1C = 2+!c. Natural or balanced
1D = 4+ suit, unbalanced
1H = 5+ !h, showing a Major
1S = 5+ !s, showing a Major
2C = 20--21 bal / Any game force
2D = 6+!h or 6+!s, 5--9 hcp
2H = 6+ suit, 10--13 hcp
2S = 6+ suit, 10--13 hcp
2N = 22--24
3C = 6+!c, Preemptive
3D = 6+!d, Preemptive
3H = 6+!h, Preemptive
3S = 6+!s, Preemptive
3N = Gambling
