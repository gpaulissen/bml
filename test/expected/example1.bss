*00{}=NYYYYYY
001C=NYYYYYY0082+!c. Natural or balanced
001D=NYYYYYY0084+ suit, unbalanced
001H=NYYYYYY0085+ suit
001S=NYYYYYY0085+ suit
002C=NYYYYYY00820--21 bal / Any game force
002D=NYYYYYY0086+!h or 6+!s, 5--9 hcp
002H=NYYYYYY0086+ suit, 10--13 hcp
002S=NYYYYYY0086+ suit, 10--13 hcp
002N=NYYYYYY022--24
003C=NYYYYYY008Preemptive
003D=NYYYYYY008Preemptive
003H=NYYYYYY008Preemptive
003S=NYYYYYY008Preemptive
003N=NYYYYYY0Gambling
