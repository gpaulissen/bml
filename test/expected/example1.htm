<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'><head><meta content="text/html; charset=utf-8" http-equiv="Content-Type" /><title>No title supplied</title><style type="text/css">div.start {
    display: inline-block;
    vertical-align: top;
    width: 60px;
}
    
div.bidtable {
    width:700px;
    padding-bottom: 20px;
}    

div.bidtable ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

div.bidtable li ul
{
    list-style-type: none;
    margin: 0;
    padding: 0;
    margin-left: 60px;
}

div.tree {
    list-style-type: none;
    padding-bottom: 20px;
}

div.tree ul {
    list-style-type: none;
    /* vline.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAKAQMAAABPHKYJAAAAA1BMVEWIiIhYZW6zAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1ggGExMZBky19AAAAAtJREFUCNdjYMAEAAAUAAHlhrBKAAAAAElFTkSuQmCC) repeat-y;
    margin: 0;
    padding: 0;
    margin-left: 10px;
}

div.tree li.node {
    margin: 0;
    padding: 0 12px;
    line-height: 20px;
    /* node.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIJYVaFGwAAAARSURBVAjXY2hgQIf/GTDFGgDSkwqATqpCHAAAAABJRU5ErkJggg==) no-repeat;
}

div.tree td.node {
    vertical-align: top;
    width: 60px;
}

div.tree li.last {
    /* lastnode.png */
    background: #fff url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIIhs+gc8AAAAQSURBVAjXY2hgQIf/GbAAAKCTBYBUjWvCAAAAAElFTkSuQmCC) no-repeat;
}

div.tree li.root {
    background: #fff none no-repeat;
}

.ccolor{color:Green}
.dcolor{color:Orange}
.hcolor{color:Red}
.scolor{color:Black}
</style></head><body><div class="bidtable"><ul><li><div class="start">&empty;</div><ul><li><div class="start">1<span class="ccolor">&clubs;</span></div>2+<span class="ccolor">&clubs;</span>. Natural or balanced</li><li><div class="start">1<span class="dcolor">&diams;</span></div>4+ suit, unbalanced</li><li><div class="start">1M</div>5+ suit</li><li><div class="start">2<span class="ccolor">&clubs;</span></div>20&ndash;21 bal / Any game force</li><li><div class="start">2<span class="dcolor">&diams;</span></div>6+<span class="hcolor">&hearts;</span> or 6+<span class="scolor">&spades;</span>, 5&ndash;9 hcp</li><li><div class="start">2<span class="hcolor">&hearts;</span><span class="scolor">&spades;</span></div>6+ suit, 10&ndash;13 hcp</li><li><div class="start">2NT</div>22&ndash;24</li><li><div class="start">3X</div>Preemptive</li><li><div class="start">3NT</div>Gambling</li></ul></li></ul></div></body></html>